public/celestia-node.sh

```diff
@@ -184,100 +157,39 @@ echo    # move to a new line
 if [ "$HAS_GO" = true ]; then
     case $REPLY in
         1)
-            # Install to GOBIN
             mkdir -p "$GOBIN"
             mv "$TEMP_DIR/celestia" "$GOBIN/"
             chmod +x "$GOBIN/celestia"
             echo "Binary moved to $GOBIN" | tee -a "$LOGFILE"
-            # Create a symbolic link in the temporary directory
-            ln -s "$GOBIN/celestia" "$TEMP_DIR/celestia"
-            echo "Symbolic link created in $TEMP_DIR" | tee -a "$LOGFILE"
-            echo ""
-            # Check if GOBIN is in PATH
-            if [[ ":$PATH:" != *":$GOBIN:"* ]]; then
-                echo "NOTE: $GOBIN is not in your PATH. You may want to add it by adding this line to your ~/.bashrc or ~/.zshrc:"
-                echo "export PATH=\$PATH:$GOBIN"
-                echo ""
-            fi
-            echo "You can now run celestia from anywhere (if $GOBIN is in your PATH)." | tee -a "$LOGFILE"
```

Comment:
This allows anyone to download and run it without having to navigate to the directory it was installed in, if in temp dir. I think we should leave this.